docs/writing-tests/index.md

```diff
@@ -8,43 +8,13 @@ testing plan](making-a-testing-plan) will help you decide what to write.
 
 There's also a load of [general guidelines](general-guidelines) that apply to all tests.
 
-```eval_rst
-.. toctree::
-   :maxdepth: 1
+## Choose Your Category
 
-   general-guidelines
-   ahem
-   assumptions
-   crashtest
-   css-metadata
-   css-user-styles
-   file-names
-   h2tests
-   lint-tool
-   making-a-testing-plan
-   manual
-   reftest-tutorial
-   reftests
-   print-reftests
-   rendering
-   server-features
-   submission-process
-   testdriver
-   testdriver-extension-tutorial
-   testharness
-   testharness-tutorial
-   tools
-   visual
-   wdspec
-   test-templates
-   github-intro
-```
-
-## Test Type
+There are various different ways of writing tests:
 
-Tests in this project use a few different approaches to verify expected
-behavior. The tests can be classified based on the way they express
-expectations:
+* [JavaScript tests (testharness.js)](testharness) are preferred and should be used where possible.
```

Comment:
I think `are preferred for testing DOM APIs and may be used for testing other features` if we want to talk about preferences.